options/posix/include/syslog.h

```diff
@@ -8,6 +8,30 @@
 extern "C" {
 #endif
 
+typedef struct {
+	char *c_name;
+	int c_val;
+} CODE;
+#define prioritynames ((CODE *)(const CODE []){ \
```

Comment:
If these come from musl, putting at least a notice might be necessary, even though this is a header.